### PR TITLE
Homepage pro PYBOTS

### PR DESCRIPTION
**Pull request by thejoeejoee**
HP bude tvořena z [README.md](README.md) za pomocí py balíčku Flask-misaka, který zvládne konvertovat markdown na HTML téměř stejně jako to dělá GH.

@tlapicka můžu poprosit o doinstalování `Flask-misaka` do prostředí pro aplikaci?
pokoušel jsem se sám, avšak py balíček `misaka` vyžaduje `python-cffi`, což je pythoní implementace `cffi`. `cffi` je modul pro volání zkompilovaného C kódu přímo z pythonu, ten ovšem při instalaci hlásí `c/_cffi_backend.c:15:17: fatal error: ffi.h: Adresář nebo soubor neexistuje`. Z této chyby jsem se dohledal až k nutnosti nainstalovat systémový balíček `libffi-dev`. Bylo by to prosím možné?

### CODE REVIEW

File: pybots/web/admin_index_view.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+from pybots.game.game_controller import game_controller
+from pybots.views.utils import form_to_kwargs
+from pybots.web.forms import ConfigurationForm
+from pybots.web.game_detail_view import GameDetailView
+
+
+class AdminIndexView(MethodView):
+    decorators = [form_to_kwargs]
+
+    def get(self):
```
Comment (thejoeejoee) [inline pybots/web/admin_index_view.py]: Codacy Issue found: [Method could be a function](https://www.codacy.com/app/thejoeejoee/pybots-server/file/2266725736/issues/source?bid=3184258&fileBranchId=3184263#l18)

File: tests/pybots/views/action_view.py [hunk 2]
```diff
@@ -34,23 +36,23 @@ def test_valid_request(self):
...
 
-            r = c.post('/action', data=dict(bot_id=bot_id, action='foobar'))
+            r = c.post(url_for('action'), data=dict(bot_id=bot_id, action='foobar'))
             self.assertEqual(r.status_code, 404, 'Request with invalid action.')
 
     def test_find_wall(self):
         with self.test_client as c:
-            bot_id = loads(c.get('/').data).get('bot_id')
+            bot_id = loads(c.get(url_for('init_game')).data).get('bot_id')
             found_wall = False
```
Comment (thejoeejoee) [inline tests/pybots/views/action_view.py]: Codacy Issue found: [Unused variable 'turn'](https://www.codacy.com/app/thejoeejoee/pybots-server/file/2266725765/issues/source?bid=3184258&fileBranchId=3184263#l54)

File: pybots/web/static/github.css [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+body > *:last-child {
+    margin-bottom: 0 !important;
+}
+
+a {
+    color: #4183C4;
+    text-decoration: none;
+}
+
+a.absent {
```
Comment (thejoeejoee) [inline pybots/web/static/github.css]: Codacy Issue found: [Element (a.absent) is overqualified, just use .absent without element name.](https://www.codacy.com/app/thejoeejoee/pybots-server/file/2266725696/issues/source?bid=3184258&fileBranchId=3184263#l23)

File: pybots/web/static/github.css [hunk 4]
```diff
@@ -0,0 +1,386 @@
...
+
+h3 {
+    font-size: 18px;
+}
+
+h4 {
+    font-size: 16px;
+}
+
+h5 {
```
Comment (thejoeejoee) [inline pybots/web/static/github.css]: Codacy Issue found: [Heading (h5) has already been defined.](https://www.codacy.com/app/thejoeejoee/pybots-server/file/2266725696/issues/source?bid=3184258&fileBranchId=3184263#l99)

File: pybots/web/static/github.css [hunk 5]
```diff
@@ -0,0 +1,386 @@
...
+    font-size: 14px;
+}
+
+h6 {
```
Comment (thejoeejoee) [inline pybots/web/static/github.css]: Codacy Issue found: [Heading (h6) has already been defined.](https://www.codacy.com/app/thejoeejoee/pybots-server/file/2266725696/issues/source?bid=3184258&fileBranchId=3184263#l103)

File: pybots/web/static/github.css [hunk 6]
```diff
@@ -0,0 +1,386 @@
...
+    background-color: white;
+    padding: 30px;
+    color: #333;
+}
+
+body > *:first-child {
+    margin-top: 0 !important;
+}
+
+body > *:last-child {
```
Comment (thejoeejoee) [inline pybots/web/static/github.css]: Codacy Issue found: [The universal selector (*) is known to be slow.](https://www.codacy.com/app/thejoeejoee/pybots-server/file/2266725696/issues/source?bid=3184258&fileBranchId=3184263#l14)

File: pybots/web/static/github.css [hunk 7]
```diff
@@ -0,0 +1,386 @@
...
+    margin-top: 0;
+    padding-top: 0;
+}
+
+body > h3:first-child, body > h4:first-child, body > h5:first-child, body > h6:first-child {
+    margin-top: 0;
+    padding-top: 0;
+}
+
+a:first-child h1, a:first-child h2, a:first-child h3, a:first-child h4, a:first-child h5, a:first-child h6 {
```
Comment (thejoeejoee) [inline pybots/web/static/github.css]: Codacy Issue found: [Heading (h3) has already been defined.](https://www.codacy.com/app/thejoeejoee/pybots-server/file/2266725696/issues/source?bid=3184258&fileBranchId=3184263#l140)

File: pybots/web/static/github.css [hunk 8]
```diff
@@ -0,0 +1,386 @@
+body {
+    font-family: Helvetica, arial, sans-serif;
+    font-size: 14px;
+    line-height: 1.6;
+    background-color: white;
+    padding: 30px;
+    color: #333;
+}
+
+body > *:first-child {
```
Comment (thejoeejoee) [inline pybots/web/static/github.css]: Codacy Issue found: [The universal selector (*) is known to be slow.](https://www.codacy.com/app/thejoeejoee/pybots-server/file/2266725696/issues/source?bid=3184258&fileBranchId=3184263#l10)

File: pybots/web/static/github.css [hunk 9]
```diff
@@ -0,0 +1,386 @@
...
+    margin-top: 0 !important;
```
Comment (thejoeejoee) [inline pybots/web/static/github.css]: Codacy Issue found: [Use of !important](https://www.codacy.com/app/thejoeejoee/pybots-server/file/2266725696/issues/source?bid=3184258&fileBranchId=3184263#l11)

File: pybots/web/static/github.css [hunk 10]
```diff
@@ -0,0 +1,386 @@
...
+a:first-child h1, a:first-child h2, a:first-child h3, a:first-child h4, a:first-child h5, a:first-child h6 {
+    margin-top: 0;
+    padding-top: 0;
+}
+
+h1 p, h2 p, h3 p, h4 p, h5 p, h6 p {
+    margin-top: 0;
+}
+
+li p.first {
```
Comment (thejoeejoee) [inline pybots/web/static/github.css]: Codacy Issue found: [Element (p.first) is overqualified, just use .first without element name.](https://www.codacy.com/app/thejoeejoee/pybots-server/file/2266725696/issues/source?bid=3184258&fileBranchId=3184263#l149)

**Comment by MarrekNozka** [comment]
aptitude install libffi-dev
pip install Flask-misaka 
pip install names

**Pull request merged**